### DISCUSSION TITLE
Freeplane does not print

### DISCUSSION THREAD

**Discussion by Karlkaefer256** [freeplane/freeplane#1401] [Help]
Linux Mint 19 as well as Mint 21
Brother MFC-J6510DW
Freeplane 1.11.6
Java 17.0.7

When I try to print a created mindmap, Freeplane simply does nothing (even no indication that mindmap has been sent to printer). An older version of Freeplane prints as required, other programs (Textmaker by Softmaker) also. Test page function of Linux Mint prints test page as required also. Any suggestions? Thanks in advance!

**Comment by beccon4** [comment]
Export the MindMap to PDF and then print ist. Sorry Freemind printing never worked properly - but never mind. ;-)
